nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExtractGrok.java

```diff
@@ -70,33 +65,51 @@
 @Tags({"grok", "log", "text", "parse", "delimit", "extract"})
 @CapabilityDescription("Evaluates one or more Grok Expressions against the content of a FlowFile, " +
     "adding the results as attributes or replacing the content of the FlowFile with a JSON " +
-    "notation of the matched content")
+    "notation of the matched content\n" +
```

Comment:
I would avoid using the \n here as this is rendered in a couple of places in HTML and we don't guarantee that it will render as expected. If we are going to explicitly indicate which library is being used, we should also be very clear that this is the implementation that is currently used and that it could change at any time without notice.

Reply:
thanks, i will use <br/> instead of \n. and add the notice.  (BTW: It is same implementation as was being used before, without credit)